src/handlers.rs

```diff
@@ -544,6 +547,41 @@ fn handle_read_file_sync(
   ))
 }
 
+fn handle_stdin_read_until_sync(
+    _d: *const DenoC,
+    base: msg::Base,
+    builder: &mut FlatBufferBuilder,
+) -> HandlerResult {
+  let msg = base.msg_as_stdin_read_until_sync().unwrap();
+  let until = msg.until();
+
+  debug!("handle_stdin_read_until_sync {}", until);
+  let mut buffer = Vec::new();
+  let stdin = io::stdin();
+  let mut stdin = stdin.lock();
+  stdin.read_until(until, &mut buffer)?;
```

Comment:
blocking on stdin read isn't viable... I would also like to get a readline-like thing working but I'm waiting until we've had another refactor of handler.rs to map them to Futures. As you might have noticed everything is blocking at the moment - we need better integration with the tokio event loop first.
Let's discuss on gitter if you want - but this is a more long term project.